sagemaker-neo-notebooks/edge/cpp-integration/tutorial.cc

```diff
@@ -0,0 +1,622 @@
+#include <chrono>
+#include <ctime>
+#include <exception>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+// 3rd-party
+#include <nlohmann/json.hpp>
+#include "npy.hpp"
+
+// aws-sdk
+#include <aws/core/Aws.h>
+#include <aws/core/client/AWSError.h>
+#include <aws/iam/IAMClient.h>
+#include <aws/iam/model/AttachRolePolicyRequest.h>
+#include <aws/iam/model/CreateRoleRequest.h>
+#include <aws/iam/model/GetRoleRequest.h>
+#include <aws/iam/model/ListRolesRequest.h>
+#include <aws/s3/S3Client.h>
+#include <aws/s3/model/CreateBucketRequest.h>
+#include <aws/s3/model/GetObjectRequest.h>
+#include <aws/s3/model/ListObjectsRequest.h>
+#include <aws/s3/model/PutObjectRequest.h>
+#include <aws/sagemaker/SageMakerClient.h>
+#include <aws/sagemaker/model/CreateCompilationJobRequest.h>
+#include <aws/sagemaker/model/DescribeCompilationJobRequest.h>
+#include <aws/sagemaker/model/Framework.h>
+#include <aws/sagemaker/model/InputConfig.h>
+
+// internal
+#include <dlr.h>
+
+using namespace std;
```

Comment:
I noticed that we use both `string` and `std::string` in this tutorial. Lets be consistent. 
- Option1: keep "using namespace std" and remove `std::` prefix below in the code
- Option2: remove "using namespace std" and use `std::` prefix.

More info on it:
https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice
https://www.geeksforgeeks.org/using-namespace-std-considered-bad-practice/

Reply:
it's removed